File: Seed.API/Models/Order.cs
using System;
using System.Collections.Generic;

namespace Seed.API.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public int EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public int CompanyId { get; set; }
        public Company Company { get; set; }
        public ICollection<OrderList> OrderLists { get; set; }
        public string CustomerNumber { get; set; }
        public string Transaction { get; set; }
        public decimal Discount { get; set; }
        public decimal Vat { get; set; }
        public decimal Subtotal { get; set; }
        public decimal GrandTotal { get; set; }
        public bool IsPaid { get; set; }
        public DateTime DateOrdered { get; set; }
        
    }
}

File: SeedApp/src/app/_models/Order.ts
import { OrderList } from './OrderList';
import { Employee } from './Employee';

export interface Order {

    OrderId: number;
    EmployeeId: number;
    Employee: Employee;
    CompanyId: number;
    OrderLists: OrderList[];
    CustomerNumber: string;
    Transaction: string;
    Discount: number;
    Vat: number;
    Subtotal: number;
    GrandTotal: number;
    IsPaid: boolean;
    DateOrdered: Date;
}
